addons/main/stringtable.xml

```diff
@@ -1473,5 +1473,13 @@
             <Portuguese>Ponta de flecha</Portuguese>
             <Japanese>矢尻</Japanese>
         </Key>
+        <Key ID="STR_dui_radar_show_speaking">
+            <English>Show icon when units speak</English>
```

Comment:
it may be good to indicate this is only about *players*, not units in general. 

how about  "show speech bubble icon for players while they're talking" ?

Reply:
It also works for remote controlled AI. So I went with a general meaning of it.